101-anf-nfs-volume/azuredeploy.parameters.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {    
```

Comment:
remove the parameters from here, for which you have defaultValues in the template